docs/src/components/sidebar.js

```diff
@@ -86,6 +86,18 @@ SidebarStyling.propTypes = {
   closeSidebar: PropTypes.func.isRequired,
 };
 
+const GALLERY_CHILD = [
```

Comment:
Adding in a static node for the gallery sidebar nav item, since it isn't autogenerated from the docs folder.